tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -349,57 +350,71 @@
+    "## Summary\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we compared three calculations for the H2 molecule ground state.  First, we produced a reference value using a classical minimum eigensolver. Then, we proceeded to run `VQE` using the Qiskit Aer `Estimator` with 1024 shots. Finally, we extracted a noise model from a backend and used it to define a new `Estimator` for noisy simulations. The results are:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 13,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Reference value: -1.85728\n",
-      "VQE on Aer qasm simulator (no noise): -1.85332\n",
-      "VQE on Aer qasm simulator (with noise): -1.80204\n",
-      "VQE on Aer qasm simulator (with noise and measurement error mitigation): -1.86029\n"
+      "VQE on Aer qasm simulator (no noise): -1.84322\n",
+      "VQE on Aer qasm simulator (with noise): -1.74645\n"
      ]
     }
    ],
    "source": [
-    "print(f'Reference value: {ref_value:.5f}')\n",
-    "print(f'VQE on Aer qasm simulator (no noise): {result.eigenvalue.real:.5f}')\n",
-    "print(f'VQE on Aer qasm simulator (with noise): {result1.eigenvalue.real:.5f}')\n",
-    "print(f'VQE on Aer qasm simulator (with noise and measurement error mitigation): {result2.eigenvalue.real:.5f}')"
+    "print(f\"Reference value: {ref_value:.5f}\")\n",
+    "print(f\"VQE on Aer qasm simulator (no noise): {result.eigenvalue.real:.5f}\")\n",
+    "print(f\"VQE on Aer qasm simulator (with noise): {result1.eigenvalue.real:.5f}\")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 11,
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "We can notice that, while the noiseless simulation's result is closer to the exact reference value, there is still some difference. This is due to the sampling noise, introduced by limiting the number of shots to 1024. A larger number of shots would decrease this sampling error and close the gap between these two values. \n",
+    "\n",
+    "As for the noise introduced by real devices (or simualted noise models), it could be tackled through a wide variety of error mitigation techniques. The [Qiskit Runtime Primitives](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/) have enabled error mitigation through the `resilience_level` option. This option is currently available for remote simulators and real backends accessed via the Runtime Primitives, you can consult [this tutorial](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/Error-Suppression-and-Error-Mitigation.html) for further information."
```

Comment:
```suggestion
    "As for the noise introduced by real devices (or simulated noise models), it could be tackled through a wide variety of error mitigation techniques. The [Qiskit Runtime Primitives](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/) have enabled error mitigation through the `resilience_level` option. This option is currently available for remote simulators and real backends accessed via the Runtime Primitives, you can consult [this tutorial](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/Error-Suppression-and-Error-Mitigation.html) for further information."
```